src/o365/yammer/commands/message/message-add.ts

```diff
@@ -0,0 +1,161 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  body: string;
+  repliedToId?: number;
+  directToUserIds?: string;
+  groupId?: number;
+  networkId?: number;
+}
+
+class YammerMessageAddCommand extends YammerCommand {
+  public get name(): string {
+    return `${commands.YAMMER_MESSAGE_ADD}`;
+  }
+
+  public get description(): string {
+    return 'Posts a Yammer network message on behalf of the current user';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.repliedToId = args.options.repliedToId !== undefined;
+    telemetryProps.directToUserIds = args.options.directToUserIds;
+    telemetryProps.groupId = args.options.groupId !== undefined;
+    telemetryProps.networkId = args.options.networkId !== undefined;
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const requestOptions: any = {
+      url: `${this.resource}/v1/messages.json`,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json;odata=nometadata'
+      },
+      json: true,
+      body: {
+        body: args.options.body,
+        replied_to_id: args.options.repliedToId,
+        direct_to_user_ids: args.options.directToUserIds,
```

Comment:
If I understand it correctly, the `direct_to_user_ids` property requires an array of integers. In its current shape however, it's not possible to pass multiple IDs, because they would be passed a string rather an array of integers. Is it something that we should take into account or am I understanding the docs incorrectly?

Reply:
Hi @waldekmastykarz,
no, you understand the docs correctly. I tried to address users with comma separated IDs and it seemed to work.

Do you want to handle it differently?

br,
Patrick

Reply:
If it works, then that's perfect. Could you please add a sample just to clarify it so that users won't get confused reading our docs and then the Yammer API docs?

Reply:
@waldekmastykarz just added an example